scalajslib/worker/0.6/src/ScalaJSWorkerImpl.scala

```diff
@@ -86,6 +93,20 @@ class ScalaJSWorkerImpl extends mill.scalajslib.api.ScalaJSWorkerApi {
     }
   }
 
+  def linkJs(
+      sources: Array[File],
+      libraries: Array[File],
+      destDirectory: File,
+      main: Option[String],
+      testBridgeInit: Boolean,
+      fullOpt: Boolean,
+      moduleKind: ModuleKind,
+      moduleSplitStyle: ModuleSplitStyle,
+      moduleInitializers: Seq[ModuleInitializer],
+      outputPatterns: OutputPatterns,
+      useECMAScript2015: Boolean
+  ): Result[File] = ???
```

Comment:
Why don't you move the `mill.api.Result.Failure("fastLinkJS/fullLinkJS is not supported in Scala.js below 1.3")` here instread of checking for the version earlier and then implement this with `???`.
This way you don't need to define any `isLinkJSSupported` target.

Reply:
Also, if you want to check the version in the `1` linker, the linker version is availably in the scalajs linker lib by calling:
```scala
org.scalajs.ir.ScalaJSVersions.current
```
I just tested and it displays correctly the used version.